apps/experiments/views/experiment.py

```diff
@@ -363,6 +363,19 @@ def extra_context(self):
             team_participant_identifiers.extend(self.object.participant_allowlist)
             team_participant_identifiers = set(team_participant_identifiers)
 
+        excluded_fields = [
+            "version_number",
+            "is_default_version",
+            "public_id",
+            "version_description",
+        ] + self.object.get_fields_to_exclude()
+        fields_changed = self.object.latest_version.compare_with_model(self.object, excluded_fields)
```

Comment:
I think here you want to follow the pattern in the create version view:
```python
working_experiment = self.object
version = working_experiment.version
if prev_version := working_experiment.latest_version:
    # Populate diffs
    version.compare(prev_version.version)
```

I think it would also be useful to make this into a method on the Experiment model so that you can call `experiment.compare_with_latest()`. Another useful method would be `experiment.has_changes()` which may be able to shortcircuit the process when it finds the first diff.